extras/parse_travis_log.py

```diff
@@ -886,11 +886,12 @@ def build_return_code(status_vera_init,
     #       It may cause false positives. Even though cppcheck is executed
     #       and all its messages can be found in the log, cppcheck messages
     #       will not cause the build to fail.
-    #       To activate cppcheck, add following line to the if statement below.
-    #       Remove also the two strings '(DEACTIVATED)' behind cppcheck in
-    #       the summary_table above.
+    #       To activate cppcheck, add following lines to the if statement
+    #       below. Remove also the two strings '(DEACTIVATED)' behind
+    #       cppcheck in the summary_table above.
     #
     #       (status_cppcheck_init is None or status_cppcheck_init) and \
+    #       (number_of_msgs_in_summary(summary_cppcheck) == 0) and \
```

Comment:
It is dangerous to have on the one side the "PERFORM_CPPCHECK=true" flag and then hide here in a comment that the check is not actually active. The `PERFORM_CPPCHECK` flag should fully control whether CPPCHECK is performed, displayed and taken into account when deciding whether the build succeeded or not (and similarly for the other checks).

I know we did it differently before, but running a check that no-one really looks at (who reads Travis logs except to find the reason of failure?) seems rather like a waste of resources. So I think off should be fully off.